### ISSUE TITLE
Install broken on macs due to patch missing on server

### ISSUE THREAD

**Issue by eraye1** [grpc/homebrew-grpc#80]
Downloading https://raw.githubusercontent.com/grpc/homebrew-grpc/master/patches/google_protobuf_4472b4a_Fixed-assignment-syntax-error.patch

curl: (22) The requested URL returned error: 404 Not Found

**Comment by stanley-cheung** [comment]
I thought we already fixed it last release in #79.

What language you are trying to install for?

**Comment by eraye1** [comment]
python

**Comment by stanley-cheung** [comment]
You may want to check out the main grpc repo README for Python: https://github.com/grpc/grpc/tree/master/src/python/grpcio. We have simplified the installation stories and for Python, I believe it can now be installed via `pip install grpcio`.

**Comment by eraye1** [comment]
@stanley-cheung that's true but I can't install the python plugin with `pip install grpcio`.  any suggestions on how to do that?

**Comment by soltanmm** [comment]
@eraye1 There's currently no way to install the gRPC Python protoc plugin via pip. That must be done from source. Installation via PyPI for major operating systems will be available soon.
